### PR TITLE
Transfer pokemon with per pokomon settings.

### PR DESCRIPTION
**Pull request by ajurna**
### Changes:
- when none meet cutoff best selected by CP \* IV.
- transfer_pokemon.json.example file added with values used by pokemon-go-bot

@OpenPoGo/maintainers

### CODE REVIEW

File: plugins/transfer_pokemon/__init__.py [hunk 1]
```diff
@@ -31,17 +32,41 @@ def log(text, color="black"):
...
+                if 'release_below_iv' not in pokemon_rules:
+                    pokemon_rules['release_below_iv'] = bot.config.pokemon_potential
+                if 'logic' not in pokemon_rules:
+                    pokemon_rules['logic'] = 'and'
+            except KeyError:
+                pokemon_rules = {
+                    'release_below_cp': bot.config.cp,
+                    'release_below_iv': bot.config.pokemon_potential,
+                    'logic': 'and',
+                }
```
Comment (wchill) [inline plugins/transfer_pokemon/__init__.py]: why do we try except here?

Comment (ajurna) [inline plugins/transfer_pokemon/__init__.py]: it checks if there is a rule for that pokemon. if there is no rule then it loads the defaults into the rule format.

Comment (wchill) [inline plugins/transfer_pokemon/__init__.py]: Can probably be done better by using `release_rules.get(pokemon_name, default)` where `default` is a default configuration.

Comment (ajurna) [inline plugins/transfer_pokemon/__init__.py]: so
release_below_cp = release_rules[pokemon_name].get('release_below_cp' , bot.config.cp)

Comment (ajurna) [inline plugins/transfer_pokemon/__init__.py]: this is now removed and changed to gets

**Comment by wchill** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/268/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by MaxLeiter** [comment]
:+1: 

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/268/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by MaxLeiter** [comment]
Merging; a PR updating the README would be much appreciated

:tada:

**Pull request merged**